website/static/js/share/utils.js

```diff
@@ -11,8 +11,10 @@ utils.onSearch = function(fb) {
     callbacks.push(fb);
 };
 
+var COLORBREWER_COLORS = [[166, 206, 227], [31, 120, 180], [178, 223, 138], [51, 160, 44], [251, 154, 153], [227, 26, 28], [253, 191, 111], [255, 127, 0], [202, 178, 214], [106, 61, 154], [255, 255, 153], [177, 89, 40]];
 var tags = ['div', 'i', 'b', 'sup', 'p', 'span', 'sub', 'bold', 'strong', 'italic', 'a', 'small'];
 
+/* Removes certain HTML tags from the source */
```

Comment:
How were the tags chosen? Is there any reason we shouldn't scrub all HTML tags (I believe `$osf.htmlDecode` will do this for you)?